integration_test/README.md

```diff
@@ -248,7 +248,7 @@ a hop. The following is sorted roughly in descending order of usefulness.
 ├── logs
 |   ├── build_and_test.txt
 |   ├── sponge_log.xml
-|   └── TestThirdPartyApps_ops-agent_debian-12_nginx
+|   └── TestThirdPartyApps_ops-agent_debian-11_nginx
 |       ├── main_log.txt
 |       ├── syslog.txt
 |       ├── logging-module.log.txt
```

Comment:
Taking https://btx.cloud.google.com/invocations/0c11e760-fc12-4fa1-b92d-edf225a36034 as an example, looks like the list of files is somewhat different these days (see https://console.cloud.google.com/storage/browser/ops-agents-public-buckets-test-logs/prod/stackdriver_agents/testing/consumer/third_party_apps/test/bullseye_x86_64/2435/20240708-102933/logs/TestThirdPartyApps_debian-cloud:debian-11_jetty):
```suggestion
|   └── TestThirdPartyApps_debian-cloud:debian-11_jetty
|       ├── VM_initialization.txt
|       ├── config.yaml.txt
|       ├── fluent_bit_main.conf.txt
|       ├── fluent_bit_metrics.txt
|       ├── fluent_bit_parser.conf.txt
|       ├── health-checks.log.txt
|       ├── journalctl_output.txt
|       ├── logging-module.log.txt
|       ├── main_log.txt
|       ├── metrics-module.log.txt
|       ├── nvidia-installer.log.txt
|       ├── otel.yaml.txt
|       ├── otel_metrics.txt
|       ├── syslog.txt
|       └── systemctl_status_for_ops_agent.txt
```
(might as well sort them, too).